packages/design/src/Colors.mdx

```diff
@@ -39,31 +39,66 @@ readability.
   background-color:var(--color-heading)"
 ></div>
 
+`--color-heading`
+
 Headings have a bold, high-contrast color to cement their hierarchy.
```

Comment:
Added swatch names to all semantic swatches, so lots of little changes in here that are best reviewed from the build preview link